testutil/testutil.go

```diff
@@ -300,7 +300,8 @@ func init() {
 					},
 				},
 				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
-					return GetHuman(p.Args["id"].(int)), nil
+					id, _ := strconv.Atoi(p.Args["id"].(string))
```

Comment:
Thanks a lot for going ahead and fixing it @agoalofalife — good call @adelowo, could you update it to:
```go
id, err := strconv.Atoi(p.Args["id"].(string))
if err != nil {
  return nil, err
}
return GetHuman(id), nil
```